src/menus/PluginMenus.cpp

```diff
@@ -1215,7 +1215,7 @@ BaseItemSharedPtr GenerateMenu()
    ( FinderScope{ findCommandHandler },
    Menu( wxT("Generate"), XXO("&Generate"),
       Section( "Manage",
-         Command( wxT("ManageGenerators"), XXO("Add / Remove Plug-ins..."),
+         Command( wxT("ManageGenerators"), XXO("Plugin Manager"),
```

Comment:
The first commit is simple but I think "..." is not supposed to be omitted from the new texts.  See the conversation at the Issue page.

Reply:
I have reintroduced the "..."

Reply:
I jumped the gun.  I have now removed "..." due to @LWinterberg comment on the issue page.